.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Feature Request
+about: Request a new feature
+labels: "enhancement"
+
+--
+<!--
+Please first search existing issues for the feature you are requesting; it may already exist as an open issue.
+-->
+
+<!-- Describe the feature. -->
```

Comment:
I think I would reword this one to cover both small additions (add a single missing field) as well as major additions (adding a whole field set).

So in terms of wording I wouldn't call this "feature request", but rather something like "schema changes and additions"?

And we could request details about what people want to propose changes to:

- context of the suggestion, why they're opening this issue
- field names
- example values for the fields
- suggested datatypes they think would be appropriate
- example events that should map to this use case

Reply:
Eventually the RFC process will have special handling for new field set additions. But I think the above will be helpful to get the ball rolling in simple cases, as well as new field set additions.

In the context of the RFCs, we're simply asking for the first things we expect to touch on in [Stages 0 and 1](https://epixa.github.io/ecs/stages.html)